mix.exs

```diff
@@ -65,7 +69,9 @@ defmodule Ret.Mixfile do
       {:secure_random, "~> 0.5"},
       {:bamboo, "~> 1.3"},
       {:bamboo_smtp, "~> 1.7"},
-      {:guardian, "~> 1.2"},
+      {:guardian, "~> 2.1.1"},
+      {:guardian_phoenix, "~> 2.0"},
```

Comment:
Updated `guardian` moved `Guardian.Phoenix.Socket` to its own library,  `guardian_phoenix`.